app/controllers/change_requests_controller.rb

```diff
@@ -154,9 +155,16 @@ def accept
       "#{@user.user_name}: [edit] #{@notebook.title}\n" \
       "Accepted change request from #{@change_request.requestor.user_name}"
 
+    # Revision Label Validation
+    if GalleryConfig.storage.track_revisions && params[:friendly_label] != ""
+      if verify_revision_label(params[:friendly_label], @notebook)
+        errors += verify_revision_label(params[:friendly_label], @notebook)
```

Comment:
Another spot where you are double calling the same function.

Reply:
Thank you. Good catch. My mistake. Fixed.